apps/admin/frontend/src/app.test.tsx

```diff
@@ -586,8 +589,14 @@ test('tabulating CVRs', async () => {
   fireEvent.click(markOfficialButton);
   getByText('Mark Unofficial Tally Results as Official Tally Results?');
   const modal = await screen.findByRole('alertdialog');
+  mockApiClient.markResultsOfficial.expectCallWith().resolves();
   fireEvent.click(within(modal).getByText('Mark Tally Results as Official'));
 
+  // since isOfficial is fetched from the memory backend, not the mock api
+  // client, have to set its results as official and also invalidate the query
+  await backend.markResultsOfficial();
+  await queryClient.invalidateQueries();
+
```

Comment:
Maybe there's a way to avoid `invalidateQueries`, but in a piecemeal transition we will have these points where we have to update the `backend` manually.